diff --git a/ports/pmdk/portfile.cmake b/ports/pmdk/portfile.cmake
index e8f1c309e..9f2d3a4a4 100644
--- a/ports/pmdk/portfile.cmake
+++ b/ports/pmdk/portfile.cmake
@@ -4,3 +4,3 @@ vcpkg_fail_port_install(ON_ARCH "arm" "x86")
 
-set(PMDK_VERSION "1.7")
+set(PMDK_VERSION "1.8")
 
@@ -9,4 +9,4 @@ vcpkg_from_github(
     REPO pmem/pmdk
-    REF bc5e309485df61c452d08367e4b13ba9dfed5071 #Commit id corresponding to the version 1.7
-    SHA512 15bee6a046746e4ab7e827bb36685bc5d9cdffdbc68ba86eb71e2c4bd84eb4fed4586c09174257bfd87ea178c8ee9865a8824842d7d1df67e0ae79ff80cf650e
+    REF 45e5a673eecd9a6302b8af6c4901b60f9a239b04 # devel-1.8: doc: fix indentation; 17.01.2020
+    SHA512 ac3e23657ac5c0f62cae5f4211aaea336330b9258a8e48315acd07b597ccc78e76b0f8bd63014f00c28b73c615255a1c69f27326f641d9590eaa3460bb2de374
     HEAD_REF master
@@ -19,3 +19,3 @@ vcpkg_build_msbuild(
     PROJECT_PATH ${SOURCE_PATH}/src/PMDK.sln
-    TARGET "Solution Items\\libpmem,Solution Items\\libpmemlog,Solution Items\\libpmemblk,Solution Items\\libpmemobj,Solution Items\\libpmempool,Solution Items\\libvmem,Solution Items\\Tools\\pmempool"
+    TARGET "Solution Items\\libpmem,Solution Items\\libpmemlog,Solution Items\\libpmemblk,Solution Items\\libpmemobj,Solution Items\\libpmempool,Solution Items\\Tools\\pmempool"
     OPTIONS /p:SRCVERSION=${PMDK_VERSION}
