From f6cd213d8db4ddc8bd0555237247662d253b1891 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C5=81ukasz=20Stolarczuk?= <lukasz.stolarczuk@intel.com>
Date: Mon, 7 Jun 2021 11:52:11 +0200
Subject: [PATCH] fix generating gcov files and turn-off verbose log

---
 codecov | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/codecov b/codecov
index b672be2..625d45c 100755
--- a/codecov
+++ b/codecov
@@ -1210,9 +1210,9 @@ then
     if [ "$ft_gcovout" = "0" ];
     then
       # suppress gcov output
-      bash -c "find $proj_root -type f -name '*.gcno' $gcov_include $gcov_ignore -exec $gcov_exe -pb $gcov_arg {} +" >/dev/null 2>&1 || true
+      bash -c "find $proj_root -type f -name '*.gcno' $gcov_include $gcov_ignore -execdir $gcov_exe -pb $gcov_arg {} \;" >/dev/null 2>&1 || true
     else
-      bash -c "find $proj_root -type f -name '*.gcno' $gcov_include $gcov_ignore -exec $gcov_exe -pb $gcov_arg {} +" || true
+      bash -c "find $proj_root -type f -name '*.gcno' $gcov_include $gcov_ignore -execdir $gcov_exe -pb $gcov_arg {} \;" || true
     fi
   else
     say "${e}==>${x} gcov disabled"
@@ -1547,7 +1547,7 @@ then
         report_len=$(wc -c < "$file")
         if [ "$report_len" -ne 0 ];
         then
-          say "    ${g}+${x} $file ${e}bytes=$(echo "$report_len" | tr -d ' ')${x}"
+          #say "    ${g}+${x} $file ${e}bytes=$(echo "$report_len" | tr -d ' ')${x}"
           # append to to upload
           _filename=$(basename "$file")
           if [ "${_filename##*.}" = 'gcov' ];
-- 
2.27.0

